src/scheduler/Analysis.py

```diff
@@ -169,7 +180,7 @@ def start_scheduling_process(self):
     def scheduler(self):
         while self.stop_condition.value == 0:
             try:
-                task = self.process_queue.get(timeout=float(self.config['ExpertSettings']['block_delay']))
+                task = self.process_queue.get(timeout=int(self.config['ExpertSettings']['block_delay']))
```

Comment:
There have been changes in master. Please merge master and reset these castings.

Reply:
done